### PR TITLE
Add branding badge for Labs

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Add the Guardian Labs details to the DCR model being passed down, things like logo URLs, titles etc. 
Big thanks to @shtukas 

## Before

Screenshot 2021-03-30 at 15 33 10

## After

Screenshot 2021-03-30 at 17 10 55

### CODE REVIEW

File: onward/app/models/MostPopularNx2.scala [hunk 1]
```diff
@@ -30,11 +31,26 @@ case class OnwardItemNx2(
...
-  implicit val onwardItemWrites = Json.writes[OnwardItemNx2]
+  implicit val brandingTypeWrites = new Writes[BrandingType] {
+    def writes(bt: BrandingType) = {
+      Json.obj(
+        "name" -> bt.name,
+      )
+    }
+  }
+
+  implicit val dimentionsWrites = Json.writes[Dimensions]
```
Comment (jfsoul) [inline onward/app/models/MostPopularNx2.scala:47]:
```suggestion
  implicit val dimensionsWrites = Json.writes[Dimensions]
```
tiny typo in the val name

File: onward/app/models/MostPopularNx2.scala [hunk 2]
```diff
@@ -74,6 +90,7 @@ object OnwardItemNx2 {
   }
 
   def contentCardToOnwardItemNx2(contentCard: ContentCard): Option[OnwardItemNx2] = {
+    print(ContentCard);
```
Comment (jfsoul) [inline onward/app/models/MostPopularNx2.scala:93]: let's remove this now! 🔥

File: onward/app/models/Series.scala [hunk 3]
```diff
@@ -26,7 +26,6 @@ case class SeriesStoriesDCR(
 )
 
 object SeriesStoriesDCR {
-  implicit val onwardItemWrites = Json.writes[OnwardItemNx2]
```
Comment (jfsoul) [inline onward/app/models/Series.scala:29]: was this just not needed?

File: onward/app/models/MostPopularNx2.scala [hunk 4]
```diff
@@ -137,6 +154,7 @@ object OnwardItemNx2 {
       kickerText = content.header.kicker.flatMap(_.properties.kickerText),
       starRating = content.card.starRating,
       avatarUrl = None,
+      branding = content.branding(Edition.defaultEdition),
```
Comment (jfsoul) [inline onward/app/models/MostPopularNx2.scala:157]:
```suggestion
      branding = content.branding(Edition(request)),
```
I think(?) we should take the edition of the request here, and conveniently it's already available!

**Review by jfsoul — APPROVED** [review]
Just some tiny tweaks

File: onward/app/models/MostPopularNx2.scala (hunk 2 above)
Comment (shtukas) [reply onward/app/models/MostPopularNx2.scala]: Done

File: onward/app/models/Series.scala (hunk 3 above)
Comment (shtukas) [reply onward/app/models/Series.scala:29]: Used to, but no longer, see chat :)

File: onward/app/models/MostPopularNx2.scala (hunk 4 above)
Comment (shtukas) [reply onward/app/models/MostPopularNx2.scala]: Done

**Pull request merged**